Gemfile.lock

```diff
@@ -249,11 +274,46 @@ GEM
       rspec-mocks (~> 3.10)
       rspec-support (~> 3.10)
     rspec-support (3.11.0)
+    rubocop (1.25.0)
+      parallel (~> 1.10)
+      parser (>= 3.1.0.0)
+      rainbow (>= 2.2.2, < 4.0)
+      regexp_parser (>= 1.8, < 3.0)
+      rexml
+      rubocop-ast (>= 1.15.1, < 2.0)
+      ruby-progressbar (~> 1.7)
+      unicode-display_width (>= 1.4.0, < 3.0)
+    rubocop-ast (1.15.1)
+      parser (>= 3.0.1.1)
+    rubocop-govuk (4.3.0)
+      rubocop (= 1.25.0)
+      rubocop-ast (= 1.15.1)
+      rubocop-rails (= 2.13.2)
+      rubocop-rake (= 0.6.0)
+      rubocop-rspec (= 2.7.0)
```

Comment:
`rubocop-rspec` is a pain!

Reply:
do we want `rubocop-govuk`

Reply:
In my opinion it is the right choice for a DfE project but we can and should tweak selected rules. We can set the new rule about `hash value`/`keyword` inference to `either` so we can acclimatise. I think I could grow to like it. 🤷🏼‍♂️ 
I do like linting the specs, plus we can tackle the offences one by one. Are there specific rule(s) you prefer? I have already taken the liberty of overriding one.